### ISSUE TITLE
Remove cross-signing feature flag

### ISSUE THREAD

**Issue by dbkr** [vector-im/riot-web#11219]
After https://github.com/vector-im/riot-web/issues/11215
After https://github.com/vector-im/riot-web/issues/11218
After https://github.com/vector-im/riot-web/issues/11213
Probably after https://github.com/vector-im/riot-web/issues/11220

We may want to also block this on some or all of the new UI.

 * Remove cross-signing labs flag
 * Always show new post sign-in flow after sign-in
 * Remove 'setup cross-signing button' from key backup section
 * Go to pub

**Comment by jryans** [comment]
For release day, I think we'll only go as far as enabling the flag, so moving this to a later phase.

**Comment by turt2live** [comment]
For now we've gone a route which turns the labs flag into a setting to get closer to what we want (https://github.com/matrix-org/matrix-react-sdk/pull/4416). https://github.com/matrix-org/matrix-react-sdk/pull/4417 is a demonstration of what it looks like to remove the setting, though is untested and will need updating for the route we're going. When removing the setting, please be sure to clean up newly-dead code.

**Comment by jryans** [comment]
It's now safe to remove this feature flag. We may add related admin controls (such as https://github.com/vector-im/riot-web/issues/13705), but this specific flag can be removed.

**Comment by jryans** [comment]
It would be best to land this work in the same release as https://github.com/vector-im/riot-web/issues/13705, so that there's some continuity of admin control for those who want it.

**Comment by neilisfragile** [comment]
(not directly blocked on design but inter dependent on #13705)
